packages/common/src/utils/arrays/reducers.ts

```diff
@@ -0,0 +1,15 @@
+export const getUnitQuantity = (
+  sum: number,
+  someEntity: {
+    numberOfPacks: number;
+    packSize: number;
+  }
+): number => {
+  return sum + someEntity.numberOfPacks * someEntity.packSize;
```

Comment:
I get the feeling that this is a calculation we have repeated all over the place in the current codebase 😉